siliconcompiler/dependencyschema.py

```diff
@@ -25,7 +29,38 @@ def __init__(self):
                 shorthelp="List of dependencies",
                 help="List of named object dependencies included via add_dep()."))
 
-        self.__deps = {}
+        schema.insert(
+            'source', 'default', 'path',
```

Comment:
I would rename 'package', 'default', 'root'  OR  'package', 'default', 'home'

Help "Package data source path" shows that package would be appropriate. Not all things pointed to will truly be source so 'source'. This is a packagaging action, so packageis more intuitive imo.

Reply:
I think root is better